test/unit/BPool/BPoolBase.sol

```diff
@@ -16,6 +16,7 @@ contract BPoolBase is Test, BConst, Utils {
   uint256 public tokenBindBalance = 100e18;
```

Comment:
i'd move this to the specific test then

Reply:
wasn't really sold on this since that'd mean repeating it for bind & unbind tests, but it isn't so badly coupled that it has to be the same amount on both, so I just repeated them with a more descriptive name for each one